docs/Developers/Useful_Links.rst

```diff
@@ -0,0 +1,41 @@
+Useful Links
+============
+
+The purpose of this document is to gather links that coala
+developers usually use throughout their work.
+
+If you ever encounter a link that helped you or that is not
+a part of the document and should be, feel free to suggest it
+by creating an issue in our issue tracker.
+
+Git-Links
+---------
+
+-   `Git Tutorial <https://try.github.io/levels/1/challenges/1>`_
+-   :doc:`Coala's Git Tutorial <Git_Basics>`
+-   `Git Commits <https://wiki.gnome.org/Git/CommitMessages>`_
+-   :doc:`Coala Commits <Writing_Good_Commits>`
+-   `Git Rebase <https://asciinema.org/a/78683>`_
+-   `Coala Git Repository <https://github.com/coala/>`_
+
+Python-Links
+------------
+
+-   `Code Style <https://www.python.org/dev/peps/pep-0008/>`_
+-   `Python Tutorial <https://docs.python.org/3/tutorial/>`_
+-   `Shorter Tutorial Version <https://www.stavros.io/tutorials/python/>`_
+
+rST-Links
+---------
+
+-   `Basic rST <http://www.sphinx-doc.org/en/latest/rest.html>`_
+-   `Syntax <http://www.sphinx-doc.org/en/stable/markup/toctree.html#toctree-directive>`_
+
+coala-Links
+-----------
+
+-   `Install coala <http://docs.coala.io/en/latest/Users/Install.html>`_
```

Comment:
you should also include http://api.coala.io/en/latest/Developers/Review.html as this points on how the review process is done